content/docs/introduction/autoscaling.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: Autoscaling
-subtitle: Learn how Neon automatically and transparently scales compute on demand
+subtitle: Learn how Neon automatically scales compute resources on demand
 enableTableOfContents: true
 ---
 
-A Beta version of Neon's _Autoscaling_ feature is now available for [Pro plan](/docs/introduction/pro-plan) users in selected regions. The Autoscaling feature dynamically adjusts the allocation of vCPU and RAM for a Neon compute endpoint in response to the current load, eliminating the need for manual intervention.
+A Beta version of Neon's _Autoscaling_ feature is now available for [Pro plan](/docs/introduction/pro-plan) users. The Autoscaling feature dynamically adjusts the amount of compute resources allocated to a Neon compute endpoint in response to the current load, eliminating the need for manual intervention.
```

Comment:
we discussed with @stepashka to remove the beta label

Reply:
Thank you. Renmoved.